app/cdap/components/AbstractWidget/SchemaEditor/RowButtons/Nullable.tsx

```diff
@@ -37,6 +37,11 @@ const NullableBase = ({ nullable, onNullable: onChange }: INullableBaseProps) =>
       }}
       disabled={typeof onChange !== 'function'}
       data-testid="schema-field-nullable-checkbox"
+      title={
+        nullable
+          ? 'Column is Nullable. Uncheck to make it Non-Nullable'
+          : 'Column is Non-Nullable. Check to make it Nullable'
```

Comment:
The strings should come from the i18n strings (from text-en.yaml file)

Reply:
Done